recipes-ni/ni-sysd-workarounds/files/syswebserv_workaround.service

```diff
@@ -0,0 +1,28 @@
+[Unit]
+SourcePath=/etc/init.d/systemWebServer
+Before=multi-user.target
```

Comment:
This is quite a list of deps, and if these are all really required, we should define a [target](https://www.freedesktop.org/software/systemd/man/latest/systemd.target.html#) instead with which to synchronize the services. But I suppose we can wait until webserver team decides how they want to write the service file.